testutil/testaudit/audit.go

```diff
@@ -0,0 +1,60 @@
+package testaudit
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+func VerifyLogs(t *testing.T, path string, cmds []string) {
+	abs, err := filepath.Abs(path)
+	require.Nil(t, err)
+	f, err := os.Open(abs)
+	require.Nil(t, err)
+
+	type log struct {
+		Msg string `json:"endpoint"`
+	}
+	logMap := make(map[string]bool)
+
+	fileScanner := bufio.NewScanner(f)
+	for fileScanner.Scan() {
+		bytes := fileScanner.Bytes()
+		l := new(log)
+		_ = json.Unmarshal(bytes, l)
+		logMap[l.Msg] = true
+	}
+	for _, m := range cmds {
+		if !logMap[m] {
+			t.Fatalf("audit logs not present for command %s", m)
+		}
+	}
+}
+
+// run manually to generate the encrypted audit log used in TestDecrypt
+func TestGenerateAuditForTestDecrypt(t *testing.T) {
```

Comment:
This is here so that in the future it is easy to generate an encrypted audit log; of course we only want to run this manually, so we should skip it

Reply:
can we mention this in the code itself.

Reply:
I already have a comment above the test - what else do you suggest?